numpy/distutils/misc_util.py

```diff
@@ -882,7 +882,7 @@ def _wildcard_get_subpackage(self, subpackage_name,
                                  caller_level = 1):
         l = subpackage_name.split('.')
         subpackage_path = njoin([self.local_path]+l)
-        dirs = [_m for _m in glob.glob(subpackage_path) if os.path.isdir(_m)]
+        dirs = [_m for _m in sorted(glob.glob(subpackage_path)) if os.path.isdir(_m)]
```

Comment:
Sort of inconsistent with what you do above. Either you should sort the outer list comprehension here, or you should always sort `glob`, even if it's unused.

My preference would be a `sorted_glob` helper function, with a comment explaining why it's necessary.